instructure/canvas-ios
# Add submission button to parent

### What's new?
- Added Submission & Rubric button to parent assignment details screen that opens the web based submission page.
- Created CSS injections to get rid of canvas menus to have a clear browsing experience.
- Added dynamic back button in a toolbar to modal web view in case user navigates away from the root page.

refs: [MBL-17747](https://instructure.atlassian.net/browse/MBL-17747)
affects: Parent, Student, Teacher
release note: Added submission feedback button to assignment details screen.

test plan:
- Go to assignment details screen.
- Tap on submission & rubric button.
- Check if the assignment's submission website loads.

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td><img src="https://github.com/user-attachments/assets/07babc74-ab67-432d-9bca-c5b0953c1245" maxHeight=500></td>
<td><img src="https://github.com/user-attachments/assets/33a9a8c8-db8b-4956-ad07-68592dfd0c03" maxHeight=500></td>
</tr>
<tr>
<td></td>
<td><img src="https://github.com/user-attachments/assets/d070e33a-c8eb-487c-8f5f-6d4b6ed35477" maxHeight=500></td>
</tr>
<tr>
<td colspan="2"><img src="https://github.com/user-attachments/assets/21564505-4360-49ad-9201-2b2e530ada7e" maxHeight=500></td>
</tr>
<tr>
<td></td>
<td><img src="https://github.com/user-attachments/assets/5d20156a-acae-43a3-871f-0add3a13be4a" maxHeight=500></td>
</tr>
</table>

## Checklist



[MBL-17747]: https://instructure.atlassian.net/browse/MBL-17747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
